pkgs/development/libraries/threejs/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildNpmPackage, fetchFromGitHub, lib, ... }:
+
+buildNpmPackage rec {
+  pname = "threejs";
+  version = "155";
+
+  src = fetchFromGitHub {
+    owner = "mrdoob";
+    repo = "three.js";
+    rev = "r${version}";
+    hash = "sha256-gtJCp+V8tdn17oDx+/Z3SP/Je/2vh6mf6T/FneJ0kpM=";
+  };
+
+  npmDepsHash = "sha256-1rJkDEwOJCc51OnX4v1pJw7WsunD3ud7UUSs7EC3uck=";
+
+  meta = with lib; {
+    description = "General purpose JavaScript 3D library";
+    homepage = "https://threejs.org";
+    license = licenses.mit;
+    maintainers = with maintainers; [ timokau ];
```

Comment:
Can you give a reference for this?

Reply:
https://github.com/NixOS/nixpkgs/pull/39405

Reply:
Instead of me this should probably be the sage team (i.e., mostly @collares and @omasanori nowadays) since I added this for sage. I agree however that it is better to remove it since sage doesn't need it anymore.